python/py-peakutils/Portfile

```diff
@@ -17,13 +17,15 @@ maintainers         {reneeotten @reneeotten} openmaintainer
 description         Peak detection utilities for 1D data
 long_description    ${description}
 
-homepage            https://peakutils.readthedocs.io
+homepage            https://bitbucket.org/lucashnegri/peakutils/
+distfiles           v${version}.tar.gz
+master_sites        https://bitbucket.org/lucashnegri/peakutils/get/
```

Comment:
this doesn't look like a good change to me: Python packages should just come from PyPI if possible and that's perfectly fine here. Also for end-user pointing the homepage to the documentation here is more useful than the Bitbucket page

Reply:
I hope you are taking this as the teachable moment that it is. Are you telling me that MacPorts magically finds Python packages in the same way that PyPi does? Sigh. This is not something that I learned from the MacPorts Guide.

Reply:
@joefowler You'll notice that most/many Python ports use the `python` PortGroup and that's where most of the magic happens. If you include that PortGroup and have the port named as `py-<portname>` then it assumes that you'll want to get the `portname` package from PyPI and sets many of the variables as `master_sites`, `distfiles` and such to the correct values. 

In this case, the package is called `PeakUtils` with capitalization (we don't use that typically for the MacPorts portname), so to make sure that it will still fills in the correct values we here explicitly set `python.rootname` (the default for `python.rootname` is set by the `python` PG and is just the part after `py-`). 

I hope that explanation is helpful, if you have questions please let us know and/or look at the (limited) description of the PortGroup in the [guide](https://guide.macports.org/chunked/reference.portgroup.html#reference.portgroup.python) or in the actual code of the [PortGroup](https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/python-1.0.tcl).

Finally, much of generating a default Portfile for a package from PyPI can be automated using `upt` and the backend developed for MacPorts. You can read about it [here](https://github.com/macports/upt-macports/) and install the package with `sudo port install upt`. In short, `upt package -f pypi -b macports <portname>` will get you started (you can write the output to your local port-tree by adding the `-o` option and/or recursively package all dependencies that are needed and not present yet by using the `-r` flag). Again as I said, it will need some manual adjustments but the skeleton is always pretty complete and very useful as a starting point.